h/presenters/annotation_json.py

```diff
@@ -11,59 +11,59 @@
 class AnnotationJSONPresenter:
     """Present an annotation in the JSON format returned by API requests."""
 
-    def __init__(self, annotation, links_service, user_service):
-        self.annotation = annotation
+    def __init__(self, links_service, user_service):
         self._links_service = links_service
         self._user_service = user_service
 
-    def asdict(self):
-        annotation = deepcopy(self.annotation.extra) or {}
+    def present(self, annotation):
+        model = deepcopy(annotation.extra) or {}
```

Comment:
model as in sql alchemy model? This seems to be whatever happens to be on `.extra`. 

I don't have any better suggestion, `data` and similar are also a bit meaningless.

Reply:
It's not an SQL alchemy model. Maybe data is better because at least it doesn't have an association?